spec/features/browsing_spec.rb

```diff
@@ -1,4 +1,4 @@
-require "rails_helper"
+require "features_helper"
```

Comment:
Did you forget to commit this file? It doesn't exist in the project.

https://travis-ci.org/rubyforgood/playtime/builds/288863689